phoenix-core/src/main/java/org/apache/phoenix/exception/SQLExceptionCode.java

```diff
@@ -464,6 +464,8 @@ public SQLException newException(SQLExceptionInfo info) {
     INSUFFICIENT_MEMORY(999, "50M01", "Unable to allocate enough memory."),
     HASH_JOIN_CACHE_NOT_FOUND(900, "HJ01", "Hash Join cache not found"),
 
+    STATS_COLLECTION_DISABLED_ON_SERVER(1401, "STS01", "Stats collection attempted but is disabled on server"),
```

Comment:
In general the error codes appear to be grouped by number/purpose.  Do we have system level error code or stats codes?  Do we want to reserve a prefix range for this 14XX?  Consider moving to a different location for sorting reasons.

Reply:
> In general the error codes appear to be grouped by number/purpose. Do we have system level error code or stats codes?

Yes thats correct, there are some conventions. I am not sure if these are followed here. Phoenix has a bunch of its own exception types not really relevant to SQL.

> Do we want to reserve a prefix range for this 14XX? 

I am not sure if we can do that and if yes, whats the way to do that. 

> Consider moving to a different location for sorting reasons

Will do

Reply:
@twdsilva Are you aware of any conventions we are trying to hold for SQL exceptions?

Reply:
I'm very confusion about the naming convention we used for the first two fields, eg., "1401" and "STS01". The last 4 sql exceptions defined are totally out of orders which mightn't be good.